R/broom_wrapper.R

```diff
@@ -433,7 +433,12 @@ prediction_binary <- function(df, threshold = 0.5, ...){
   }
 
   # get actual value column name from model formula
-  actual_col <- all.vars(first_model$formula)[[1]]
+  actual_col <- if(!is.null(first_model$formula)) {
```

Comment:
This function is prediction_binary and this is supposed to be from glm or xgboost and what I need is left hand side, not terms. I'll add a validation for the class though